spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -340,10 +340,13 @@ def not_ok?; false; end
       end
 
       it 'supports a new expect-based syntax' do
-        group = RSpec.describe([1, 2, 3]) do
-          it { is_expected.to be_an Array }
-          it { is_expected.not_to include 4 }
-        end
+        group =
+          with_isolated_stderr do
```

Comment:
You're welcome to rewrite/remove these after this change.

Reply:
Do you have something like
```ruby
module = Module.new do
  class << self
    def math?
      true
    end
  end
end

RSpec.describe module do
  it { is_expected.to be_a Module }
  it { is_expected.to be_math }
end
```
or
```ruby
RSpec.describe "hello" do
  it { is_expected.to be_a String }
  it { is_expected.not_to include "heaven" }
end
```
in mind?

Reply:
A class maybe?

Reply:
I was trying to play with "math class" initially, with "module" it's indeed less funny.